### PR TITLE
Enhance file right click commands and support multi projects

### PR DESCRIPTION
**Pull request by sophyallen**
## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #25

## What is the new behavior?

- Support commands via Explorer file(Move.toml, .move) right click, even the file in subproject or subfolder
- Multi project support

### CODE REVIEW

**Review by sophyallen — COMMENTED** [review]
Check done

**Comment by sophyallen** [comment]
@yubing744 All check tests has been passed, can you take the time to review?

**Comment by yubing744** [comment]
> @yubing744 All check tests has been passed, can you take the time to review?

OK

File: src/commands/mpm_commands.ts [hunk 1]
```diff
@@ -280,40 +290,63 @@ export const mpmTestFunction = (ctx: IDEExtensionContext) => {
...
+ * @param uri
+ * @returns fsPath
+ */
+function getFileDir(uri: vscode.Uri): string | undefined {
+  if (!uri) {
+    return undefined;
+  }
+  // if file uri, return the closest folder path
+  if (uri.scheme === 'file') {
+    return vscode.Uri.joinPath(uri, '../').fsPath;
```
Comment (yubing744) [inline src/commands/mpm_commands.ts:348]: 这里最好递归往上找  Move.toml 文件作为项目目录，直到工作空间的根目录

**Comment by yubing744** [comment]
集成测试也需要添加一个多项目的测试用例

**Review by yubing744 — APPROVED** [review]
I tested it locally, the experience is good, I merged

**Pull request merged**